### ISSUE TITLE
KITTI reproduce

### ISSUE THREAD

**Issue by RuiTianHIT** [Lavreniuk/EVP#11]
Great work!  
We want to reproduce the very goog work in KITTI dataset.  am very interested to try it out !

However, in the readme, I don’t seem to see the installation process and the pre-trained pth file. For the KITTI data set, could you please help us? Thank you very much and look forward to your reply.

**Comment by RuiTianHIT** [comment]
such as nyu_class_embeddings_my_captions.pth, 
in the KITTI dataset, the "kitti_class_embeddings_my_captions.pth" is missed.

**Comment by RuiTianHIT** [comment]
@Lavreniuk Thank you very much and look forward to your reply.

**Comment by Lavreniuk** [comment]
hi, @RuiTianHIT , thank you for interesting in our work.
I have added the missing file: https://huggingface.co/spaces/MykolaL/evp/tree/main/depth
If you find our work useful, we will appreciate your likes in git repo as well as in huggingface :)

**Comment by RuiTianHIT** [comment]
@Lavreniuk 
Hi, we thank you for the reply very much!!!
We see the "[depth.md](https://github.com/Lavreniuk/EVP/blob/main/depth/README.md)" for NYUDepthV2 datasets. 
We hope you can provide some directory of KITTI dataset in your project, installation details. We believe this will facilitate our reproduction of the results on the KIITI data set.

Thank you again for the author's help. I believe good results can be achieved through your help!!!

**Comment by Lavreniuk** [comment]
@RuiTianHIT , I have used standard pipe line for KITTI data preparation, you could found one that I followed here:
https://github.com/cleinc/bts/tree/master/pytorch#kitti

**Comment by RuiTianHIT** [comment]
@Lavreniuk Thank you very much. Recently, I want to reproduce the high-quality work on the KITTI dataset, could you please to share the "best_model_kitti.ckpt". 

That can be of great help to me!!!
Once again, thank you to the author!!!
Looking forward to your reply!!!

**Comment by RuiTianHIT** [comment]
@Lavreniuk  i have 2 questions in the reproduce process.
1. ""Some weights of the model checkpoint at openai/clip-vit-large-patch14 were not used when initializing CLIPTextModel:[ .... ]""
Does this indicate that there is an issue with the loading weight process? I would like to know how I can solve it .

2. During the training phase, I find a print in the terminal, \ie 
"" of train images: 23158
<class 'dataset.kitti.kitti'>
Dataset: KITTI of test images: 652""

This seems to be inconsistent with what is stated in the paper!

![question](https://github.com/Lavreniuk/EVP/assets/104761701/24f91382-578a-4192-bdaa-031221e69ecc)

**Comment by Lavreniuk** [comment]
@RuiTianHIT ,
according to weights in OpenAI VIT model - it is ok. Don't worry. 
according to the image list, so there are 697 images in common test set, however not each has proper depth map.
please check here (you could see None in some places):
https://github.com/hisfog/SfMNeXt-Impl/blob/3322a37f759fe9b14d9c11d868e83119ac37c203/finetune/train_test_inputs/kitti_eigen_test_files_with_gt.txt

Thus normal pairs are slightly less than all images (both test and train).
so you could check list of proper pairs in my repo, or in some others, like this:
https://github.com/wl-zhao/VPD/tree/main/depth/dataset/filenames/eigen_benchmark

so 652 is right number of proper pairs! but total images are 697 in test set.
